src/ui/room_properties.ui

```diff
@@ -227,16 +236,25 @@
      </layout>
     </widget>
    </item>
-   <item row="2" column="0" colspan="2">
+   <item>
     <widget class="QGroupBox" name="groupBox_pathfinding">
      <property name="title">
       <string>Pathfinding</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_pathfinding">
+      <property name="sizeConstraint">
+       <enum>QLayout::SetMinimumSize</enum>
+      </property>
       <item>
        <widget class="QLabel" name="label_weightInstructions">
+        <property name="sizePolicy">
+         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
+          <horstretch>0</horstretch>
+          <verstretch>0</verstretch>
+         </sizepolicy>
+        </property>
         <property name="text">
-         <string>Enter new roomweight (= travel time), minimum is 1:</string>
+         <string notr="true">&lt;p&gt;Dummy text replaced when used.&lt;/p&gt;</string>
```

Comment:
```suggestion
         <string notr="true">Dummy text replaced when used</string>
```

Reply:
Tooltips need those paragraph markers to word-wrap - fair enough no-one other than someone using the preview option in Qt Designer is going to see this but I keep them in for consistency...